src/Core/Framework/DataAbstractionLayer/FieldSerializer/EmailFieldSerializer.php

```diff
@@ -34,6 +36,10 @@ public function decode(Field $field, $value)
 
     protected function getConstraints(Field $field): array
     {
-        return [new Email()];
+        return [
+            new NotBlank(),
```

Comment:
The Email Serializer should not restrict that

Reply:
Is removed 👍

Reply:
Can you test that you can make an Email field nullable?

Reply:
As discussed via slack: The changes in this file are completely reverted